org-formation/600-access/_tasks.yaml

```diff
@@ -430,3 +430,35 @@ SynapseLlmDeveloperPolicy:
         ]
       }
     PolicyName: SynapseLlmDeveloperPolicy
+
+SynapseLlmLambdaExecRoleBoundary:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.5.1/templates/IAM/managed-policy.yaml
+  StackName: synllm-lambdaexecrole-boundary
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
```

Comment:
if you intention is to deploy only to synapse prod account then this should be `false`